internal/provider/servicenow/api.go

```diff
@@ -118,6 +119,10 @@ func (snp *ServiceNowProvider) getRecords(
 		sysparmQuery,
 	)
 
+	if extraQueryParms != "" {
+		url = url+ extraQueryParms
+	}
```

Comment:
As stated in another comment, this won't be necessary if a helper func like `buildOptionalParams` is used to build the encoded query param string and set the params in the [fmt.Sprintf](https://github.com/newrelic/nr-entity-tag-sync/blob/9c2ba6f3b7db03189073ac53a82a0c004e15367b/internal/provider/servicenow/api.go#L112-L119) that comes right before this.

Reply:
done